com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -92,6 +93,12 @@ public PostProcessPass(RenderPassEvent evt, PostProcessData data, Material blitM
                 m_UseRGBM = true;
             }
 
+            // Only two components are needed for edge render texture, but on some vendors four components may be faster.
+            if (SystemInfo.graphicsDeviceVendor.ToLowerInvariant().Contains("arm"))
```

Comment:
Maybe also check if format is supported:
```suggestion
            if (SystemInfo.IsFormatSupported(GraphicsFormat.R8G8_UNorm) && SystemInfo.graphicsDeviceVendor.ToLowerInvariant().Contains("arm"))
```

Reply:
Done